draft-irtf-cfrg-ristretto255-decaf448.md

```diff
@@ -233,23 +235,25 @@ The only operations exposed by each abstract group are decoding,
 encoding, equality, a one-way map, addition, negation, and the
 derived subtraction and (multi-)scalar multiplication.
 
-Decoding is a function from bytestrings to abstract elements with
+Decoding is a function from byte strings to abstract elements with
 built-in validation, so that only the canonical encodings of valid
 elements are accepted. The built-in validation avoids the need for
 explicit invalid curve checks.
 
-Encoding is a function from abstract elements to bytestrings so that all
+Encoding is a function from abstract elements to byte strings so that all
 equivalent representations of the same element are encoded as identical
-bytestrings. Decoding the output of the encoding function always
+byte strings. Decoding the output of the encoding function always
 succeeds and returns an equivalent element to the encoding input.
 
 The equality check reports whether two representations of an abstract
 element are equivalent.
 
-The one-way map is a function from uniformly distributed bytestrings
+The one-way map is a function from uniformly distributed byte strings
 of a fixed length to uniformly distributed abstract elements. This map
-is not invertible and is suitable for hash-to-group operations and to
-select random elements.
+is suitable for hash-to-group operations and to select random elements.
+The map is not intended to be invertible, but its one-way nature alone
+should not be relied on for security, as it's possible to find valid
+inputs for a given output.
```

Comment:
```suggestion
The map is injective, meaning that it maps each input to a distinct 
output, though not pre-image resistant, meaning one can find a valid
input for a given output. As such, this function MUST NOT be used in
settings that require pre-image resistance.
```

It seems better to just use the technical term 🤷

Reply:
Agreed on "not pre-image resistant", but injective loses the focus on the primary implication, which is that there is no map from point to uniform string. (It technically does mean that, but the average implementer might not know.) The MUST NOT can't stand unqualified, because we do expect it to be used in settings that require pre-image resistance when paired with a hash function.

```suggestion
The map is not invertible, but also not pre-image resistant,
meaning an attacker can find a valid input for a given output.
```